fuel-core-interfaces/src/db.rs

```diff
@@ -146,7 +147,7 @@ pub mod helpers {
     pub struct Data {
         /// Used for Storage<Address, (u64, Option<Address>)>
         /// Contains test validator set: <validator_address, (stake and consensus_key)>;
-        pub validators: HashMap<Address, (ValidatorStake, Option<Address>)>,
+        pub validators: HashMap<ValidatorAddress, (ValidatorStake, Option<ConsensusPublicKey>)>,
```

Comment:
Probably want to update the docstrings for this and other places the name was changed.